lib/models/builder.js

```diff
@@ -36,8 +36,21 @@ module.exports = Task.extend({
     this.trapSignals();
   },
 
+  /**
+    Determine whether the output path is safe to delete. If the outputPath
+    appears anywhere in the parents of the project root, the build would 
+    delete the project directory. In this case return `false`, otherwise
+    return `true`.
+  */
   canDeleteOutputPath: function(outputPath) {
-    return this.project.root.indexOf(outputPath) === -1;
+    var rootPathParents = [this.project.root];
+    var parsedPath = path.parse(this.project.root);
```

Comment:
Yup, didn't realize that until I saw the failure this morning. Can probably rework using path.dirname() and path.sep instead. Plan to look at that either tonight or tomorrow.